### ISSUE TITLE
How to specific the result directory to something other than boofuzz-results?

### ISSUE THREAD

**Issue by prRZ5F4LXZ** [jtpereyda/boofuzz#502]
```
boofuzz/constants.py
7:RESULTS_DIR = "boofuzz-results"
```

I see "boofuzz-results" is hard-code in the source code. Is there a way to specify where to save the file?

**Comment by SR4ven** [comment]
Right now it seems to be fully hard coded.

I guess we could see this as a feature request? Shouldn't be too hard to implement an additional Session argument to specify the database file path.

**Comment by jtpereyda** [comment]
Yep, I figure a `-o` option to specify the directory name. Right now "boofuzz-results" is a directory in which results directories are  created. I'm leaning toward following AFL's example and specifying the entire output directory names. E.g. `-o boofuzz-results/testRun1` etc.

**Comment by SR4ven** [comment]
The session argument `db_filename` was implemented in #588.

A command line argument like `-o` still has to be added to boofuzz/boofuzz/cli.py.
